yorkie.rb

```
@@ -0,0 +1,58 @@
+  end
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = %W[
+      -s -w
+      -X github.com/yorkie-team/yorkie/internal/version.Version=#{version}
+      -X github.com/yorkie-team/yorkie/internal/version.BuildDate=#{time.iso8601}
+    ]
+
+    system "go", "build", *std_go_args(ldflags:), "./cmd/yorkie"
```

RuboCop (1.75.5)

[fatal] unexpected token tRPAREN
(Using Ruby 2.7 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)